### PR TITLE
s3ForcePathStyle no longer the correct name for techdocs S3 publisher

### PR DESCRIPTION
**Pull request by dhhuynh2**
## Hey, I just made a Pull Request!

techdocs-node [was recently bumped to use AWS SDK v3 ](https://github.com/backstage/backstage/commit/37931c33ce#), however, this appears to have broken use of `s3ForcePathStyle`, meaning that people who use s3 and reference their buckets path-style have broken techdoc usage. This was recently changed in AWS SDK v3 to be `forcePathStyle` instead, [as detailed in this issue](https://github.com/backstage/backstage/issues/15605).

The simplest fix would be to correct this to `forcePathStyle` in the awsS3 publishing step for techdocs-node. I would prefer to change the config definition as well, but it seems that there are parts of backstage that are still using v2 (backend-common, catalog-backend-module-aws), as documented in this issue:

https://github.com/backstage/backstage/issues/15516

Changing the config definition from `s3ForcePathStyle` to `forcePathStyle` seems like it'd necessitate an upgrade of all instances of the v2 SDK to v3, which is probably larger in scope. Feel free to let me know if it's preferable to look at that first, or if we'd like to get this short-term fix in first. 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/cyan-deers-walk.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-node': minor
```
Comment (agentbellnorm) [inline .changeset/cyan-deers-walk.md:2]: Should this be a patch?

**Review by agentbellnorm — COMMENTED** [review]
Thanks for the PR! I think the change makes sense given that `techdocs-node` already uses v3. I had one small question.

File: .changeset/cyan-deers-walk.md (hunk 1 above)
Comment (dhhuynh2) [reply .changeset/cyan-deers-walk.md]: Yep, makes sense to me, thanks.

https://github.com/backstage/backstage/pull/15613/commits/3c64e308780e0516586e4309d6a217f72dd7cb58

**Review by agentbellnorm — APPROVED** [review]

**Comment by benlangfeld** [comment]
Hey @agentbellnorm , since this is a bugfix, would it not make more sense for it to go in 1.10.2, or might it otherwise be possible to cut a patch release of this plugin before the next backstage release? 14th Feb is quite some time away and at least for us this bug falls into the "high impact" criteria for an [emergency release](https://github.com/backstage/backstage/blob/master/docs/publishing.md#emergency-release-process).

**Comment by bralandealmeida** [comment]
> Hey @agentbellnorm , since this is a bugfix, would it not make more sense for it to go in 1.10.2, or might it otherwise be possible to cut a patch release of this plugin before the next backstage release? 14th Feb is quite some time away and at least for us this bug falls into the "high impact" criteria for an [emergency release](https://github.com/backstage/backstage/blob/master/docs/publishing.md#emergency-release-process).

In addition, I'm waiting for this merge to be able to implement it in the company

**Pull request merged**